runtime/doc/news.txt

```diff
@@ -75,6 +75,8 @@ LSP
   customizing the transformation of an LSP CompletionItem to |complete-items|.
 • |vim.lsp.diagnostic.from()| can be used to convert a list of
   |vim.Diagnostic| objects into their LSP diagnostic representation.
+• |vim.lsp.buf.code_action()| actions show client name when there are multiple
+  clients.
```

Comment:
this isn't a breaking change, please move it to the "features" LSP section below